components/layout_2020/fragment_tree/fragment.rs

```diff
@@ -178,6 +178,15 @@ impl Fragment {
         }
     }
 
+    pub fn scrolling_area(&self, containing_block: &PhysicalRect<Length>) -> PhysicalRect<Length> {
+        match self {
+            Fragment::Box(fragment) | Fragment::Float(fragment) => fragment
+                .scrollable_overflow(&containing_block)
```

Comment:
Do we need to re-borrow `containing_block` here? `fn scrollable_overflow` takes a `&PhysicalRect`, so I think we should be able to pass `containing_block` directly?

Reply:
Nope. You're right. I've removed this.